### DISCUSSION TITLE
Dear developers, please give some love to visual content creators... Presentation of images.

### DISCUSSION THREAD

**Discussion by interpixels** [mastodon/mastodon#23140] [General]
I have been using Mastodon for about 1 month, and so far, my experience has been wonderful, except one area: the presentation of images. There has been a lot of talk recently about Mastodon as a replacement for Twitter, but I think Mastodon has the potential to combine the roles that are currently served by Twitter and Instagram.

But unfortunately, the current treatment of images on the website (desktop/mobile) is brutal and cruel, in my opinion. I see 3 main issues: 

1. the default crop of 16:9 doesn't make much sense. Switching it off is possible only for logged-in users and many people don't realize that such an option exists. Yes, I know there is an issue already here:
https://github.com/mastodon/mastodon/issues/18686#issuecomment-1385629342

2. the eye icon on top of images - as far as I know there is no way for the user to disable it. At first, I tried to pretend it is not there... But after some time, it is no longer possible to pretend. I find it extremely irritating that it covers a part of the image. Especially on mobiles, as its size relative to the image size is much bigger than on desktops. I don't know how many people use it on a regular basis. 3, 5 or 7? I don't understand the point of this function, using it won't cause me to "unsee" an image... But OK, why not have it as an option rather than spoiling the visual experience for everyone?

3. this one is very subjective: rounded corners of images. I realize that many people may like them and I accept it. I just have a visual allergy against them. I feel like they're "damaging" the artwork; something is bitten off, taken away, it just feels imperfect and childish. Can you imagine a museum presenting the paintings of Rembrandt or van Gogh with rounded corners, "because some people find it cool"? It would be great if one could deactivate the rounded corners in the settings...

Overall, I am very excited about Mastodon, the whole concept is great. And I also think the web interface is just fantastic, with the exception of how images are presented. It really hurts my eyes and brain. I think art should be presented on Mastodon in the way it was created by the artist (just scaled down of course) - that is without any distortions, crops or things put on top of it. 

Just my two cents... Greetings :)

**Comment by DrCrlsn** [comment]
I agree that visual content creators have a lot riding on how Mastodon displays their work.

I'd also like to see consistency in how profile header images are displayed in Mastodon web and mobile (android) apps. Unexpanded profile header image (i.e., image not viewed in new window) seems to be cropped three different ways: top-bottom in desktop web app, left-right in mobile web app, and left-right *and* top-bottom in mobile standalone app. None of these ways results in an image with 3:1 aspect ratio. Thanks.
